sound/soc/codecs/rt715.c

```diff
@@ -855,6 +853,11 @@ int rt715_io_init(struct device *dev, struct sdw_slave *slave)
 	/* Finish Initial Settings, set power to D3 */
 	regmap_write(rt715->regmap, RT715_SET_AUDIO_POWER_STATE, AC_PWRST_D3);
 
+	if (rt715->first_hw_init)
+		regcache_mark_dirty(rt715->regmap);
```

Comment:
question: why doesn't the rt715 driver use the same tricks as the other drivers?

````c
if (rt1308->first_hw_init) {
		regcache_cache_only(rt1308->regmap, false);
		regcache_cache_bypass(rt1308->regmap, true);
	}

if (rt1308->first_hw_init) {
                 regcache_cache_bypass(rt1308->regmap, false);
	         regcache_mark_dirty(rt1308->regmap);
````

Reply:
The io_init function of rt711/1308 has the necessary process to do like the calibration and EFUSE reading. Therefore, the cache_only set to false and cache_bypass set to true, the io_init function will complete the process fully.
But, the io_init function in rt715 driver doesn't have such the HW process, these settings are the default settings. I think the rt715 doesn't need to use the same tricks.
Regarding to the rt5682, the rt5682 has the calibration process to do. So, the rt5682 codec driver will to the same tricks for it. I told to oder_chiou already, he will modify the driver.